raiden/tests/utils/smoketest.py

```diff
@@ -306,7 +307,7 @@ def setup_matrix_for_smoketest(
     print_step: Callable,
     free_port_generator: Iterable[Port],
     broadcast_rooms_aliases: Iterable[str],
-) -> Iterator[List[Tuple["ParsedURL", HTTPExecutor]]]:
+) -> Generator[List[Tuple["ParsedURL", HTTPExecutor]], None, None]:
```

Comment:
Why the change to Generator?

Reply:
Ah that's actually unnecessary. 

I had some "fun" with mypy and at some point it complained that the return type should be `Generator[...]` or one of it's supertypes. But Iterator is a super-type of Generator.

I'll revert that part.